NSS-Day-Cohort-53/giffygram-spazzy-kermits
# notifications work as written in the issues. Message list updates at …

…appropriate times

#### Changes Made
1. Added new function in msgList that updates the read of all messages that are unread currently rendered. Sends out notification update event.
2. Read function is called on msgOpen.
3. MsgList is rerendered on msgClose, user does not see it since it is shut.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    >when user clicks notification. Msgsidebar opens and notifications are cleared. 
    > when user closes msgsidebar. Msglist is rerendered with now read messages no longer marked.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
